examples/IBFE/explicit/ex0/example.cpp

```diff
@@ -230,7 +230,10 @@ bool run_example(int argc, char** argv, std::vector<double>& u_err, std::vector<
             new IBFEMethod("IBFEMethod",
                            app_initializer->getComponentDatabase("IBFEMethod"),
                            &mesh,
-                           app_initializer->getComponentDatabase("GriddingAlgorithm")->getInteger("max_levels"));
+                           app_initializer->getComponentDatabase("GriddingAlgorithm")->getInteger("max_levels"),
+                           /*register_for_restart*/ true,
+                           app_initializer->getRestartDumpDirectory(),
+                           app_initializer->getRestartRestoreNumber());
```

Comment:
The changes to this file are only necessary to update this example to use restart data, right?

Reply:
Yes; without them, `IBFEMethod` does not read in the libMesh data stored in the XDA/XDR files. It would be nice to embed this information in the `IBFEMethod` restart database entry; unfortunately, the `putToDatabase` interface in SAMRAI is designed to be oblivious to this kind of information.

Reply:
Ah, so one thing that we could do is to make a derived class from `SAMRAI::tbox::RestartManager` that includes hooks for handling this "extra" restart data, via `RestartManager::registerSingletonSubclassInstance()`.

That is beyond the scope of this PR, however.

Reply:
That would be very nice, and another project for a student who wants to do some nontrivial modifications to IBAMR :)